samples/net/sockets/http_get/sample.yaml

```diff
@@ -15,6 +15,7 @@ tests:
     # CONFIG_NET_SOCKETS_POSIX_NAMES.
   sample.net.sockets.http_get.posix:
     filter: not CONFIG_NET_SOCKETS_OFFLOAD
+    platform_exclude: cc3220sf_launchxl cc3235sf_launchxl
```

Comment:
Do both of these platforms use socket offloading? Do you have an idea why preceding line of `filter: not CONFIG_NET_SOCKETS_OFFLOAD` doesn't catch them?

Reply:
Yes both of the platforms use socket offloading.

I do not fully understand the filtering system, but I have noticed that the build reports an error about the conflict with the Kconfig options prior to applying the filter. So for instance, if I modify sample.yaml to

```
  sample.net.sockets.http_get.posix:
    filter: not CONFIG_NET_SOCKETS_OFFLOAD
    extra_configs:
      - CONFIG_NET_SOCKETS_POSIX_NAMES=y
```

then there is no issue in Kconfig, and the test would be correctly skipped, since CONFIG_NET_SOCKETS_OFFLOAD=y is set. It seems inherent to the system to apply the filters at a late stage after Kconfig is done.

Reply:
> but I have noticed that the build reports an error about the conflict with the Kconfig options prior to applying the filter

I see. As we really need to fix this issue soon, let's go with the current solution.